doc/api/url.markdown

```diff
@@ -66,6 +70,10 @@ string will not be in the parsed object. Examples are shown for the URL
 
     Example: `'#hash'`
 
+* `auth`: The authentication information portion of a URL.
+
+    Deprecated: `auth` does not handled URL encoding properly, use `username` and `password` instead.
```

Comment:
I really don't think this should be deprecated. Mainly because the user should be in control if they should need to parse `auth` differently than `url.parse()`. For example, maybe they want to split on the middle-most colon or need access to the raw representation (without decoding). Without this property, they'd have no way to do that.